src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
-body {
+// #region head
+body,
+h1,
+h2,
+p {
   margin: 0;
+  font-family: "Roboto", "Open Sans", sans-serif;
+  color: #060b35;
 }
```

Comment:
The main font-family should be applied to the html or body tag.
For color, as you need it in a few places let's create a variable and use it.